app/scripts/modules/kubernetes/src/v2/pipelines/stages/runJob/KubernetesV2RunJobStageConfig.tsx

```diff
@@ -43,9 +61,94 @@ export class KubernetesV2RunJobStageConfig extends React.Component<IStageConfigP
     this.initRawManifest();
   }
 
+  public sourceChanged = (event: any) => {
```

Comment:
I think all these public methods can be private?

Reply:
yes :)